src/ModelList.js

```diff
@@ -0,0 +1,354 @@
+import { anim } from 'modapp-utils';
+import { RootElem } from 'modapp-base-component';
+
+/**
+ * A component rendering a list of key/value pairs based on a model
+ */
+class ModelList extends RootElem {
+
+	/**
+     * Creates an instance of ModelList
```

Comment:
Indentation: JSDoc comments indentation should be tab-indented and space-aligned (just like in Go :) ):  `<tab><space>` instead of `<space> * 5`.